ArduCopter/mode_plancktracking.cpp

```diff
@@ -37,20 +37,57 @@ void ModePlanckTracking::run() {
 
     // check if gimbal steering needs to controlling the vehicle yaw
     AP_Mount *mount = AP::mount();
-    if ((mount != nullptr) && (mount->mount_yaw_follow_mode == AP_Mount::vehicle_yaw_follows_gimbal)) {
-        float angle_deg = mount->get_follow_yaw_rate();
-        int8_t direction = 1;
-        bool relative_angle = true;
-        if (angle_deg < 0.0f) {
-            angle_deg = -angle_deg;
-            direction = -1;
+    bool paylod_yaw_rate = false;
+    float yaw_rate_for_logging_cds=18.4218;
+    if(mount != nullptr) {
+      if(mount->mount_yaw_follow_mode == AP_Mount::vehicle_yaw_follows_gimbal) {
```

Comment:
Do we need to make sure that the gimbal yaw is ignored during takeoff & landing? Or is that already accounted for elsewhere?

Reply:
That should already be accounted for, but I'll review to be sure. I know we discussed this at the time, and put in the appropriate measures for land, but I don't remember for sure about takeoff

Reply:
I looked through the code and I don't see anything that would limit the yaw control during takeoff & landing